pkgs/by-name/re/recaf/package.nix

```diff
@@ -0,0 +1,108 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  git,
+  jdk23,
+  gradle,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+}:
+let
+  pname = "recaf";
+  jdk = jdk23.override { enableJavaFX = true; };
+in
+stdenv.mkDerivation {
+  inherit pname;
+  version = "4.0-unstable-2024-12-11";
+
+  src = fetchFromGitHub {
+    owner = "Col-E";
+    repo = "Recaf";
+    rev = "483b8004251e5c654090d2315af30913d724a083";
+    hash = "sha256-OKz9KSl1g0ddLm38f6E6Ee9X0Q53hngPZPFDtramunQ=";
+    #Required for build info
+    leaveDotGit = true;
```

Comment:
```suggestion
```
This needs to be patched out of the gradle scripts.
The hash is not reproducible.